shapeshift/web
# feat: disable savers/withdraw DefiAction buttons while Txs are pending

## Description

Does what it says on the box, ensures users aren't getting rugged by depositing and encountering a race condition we don't know about, using another address or anything else that could make funds "stuck".

Tooltip copies temporary and not approved by @shapeshift/product 

## Notice




## Pull Request Type


## Issue (if applicable)

N/A

## Risk



## Testing



- Deposit into UTXOs (longer time to confirmation, easier to test)
- Ensure the action buttons are disabled in overview and a tooltip is shown before the first confirmation
- Ensure the action buttons are re-enabled on Tx completion (1 confirmation on-chain)

### Engineering



- Implementation looks sane

### Operations



- ☝🏽 

## Screenshots (if applicable)

<img width="541" alt="image" src="https://user-images.githubusercontent.com/17035424/215540582-288c81c6-9f0b-4573-ba03-a869ca0557cd.png">
<img width="523" alt="image" src="https://user-images.githubusercontent.com/17035424/215540621-6b21ea24-48c3-4a3b-be79-49a6ee18508c.png">
<img width="537" alt="image" src="https://user-images.githubusercontent.com/17035424/215540659-985b2359-46fb-402a-8375-68cdddc33ce5.png">
